daemon/swap_daemon_erc20_test.go

```diff
@@ -35,7 +35,7 @@ func TestRunSwapDaemon_ExchangesXMRForERC20Tokens(t *testing.T) {
 	tokenAsset := types.EthAsset(tokenAddr)
 
 	timeout := 7 * time.Minute
-	ctx := LaunchDaemons(t, timeout, aliceConf, bobConf)
+	ctx, _ := LaunchDaemons(t, timeout, aliceConf, bobConf)
```

Comment:
I had to change the API to `LaunchDaemons`. You can just ignore the changes outside of `daemon/issue385_test.go`.